Mathlib/MeasureTheory/Measure/Trim.lean

```diff
@@ -0,0 +1,136 @@
+/-
+Copyright (c) 2023 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
```

Comment:
Is this correct? I thought we've had `trim` for much longer than 9 months

Reply:
I was unsure whether the date should reflect the date of creation of the content, or the date of creation of the new file. I'll change it to 2021.